process faults {5: ({6, 4, 5, 2}, 2794391717)}
network faults {2: [{0}, {5, 1, 6}, {3, 2, 4}], 6: [{0, 1}, {3, 2}, {4}, {5, 6}]}
{5: ("53450160B99B4C591A2BCC21E94A8787733F02F164855DF729C0CB0E9DD0C7BE", {6, 4, 0, 2, 5, 3, 1}), 9: ("37039302AF70204E4E72040A7AFE3B4AC7FBFA9C1B67917617E65DC1C479FA77", {4, 3, 1, 5, 0, 6, 2}), 6: ("C90F5821B585BB9FBDE48645F9A81275217DD02A2F2F9EAA7B3540D10F41CB34", {0, 4, 5, 3, 1, 6, 2}), 10: ("F7503261871857DABAB062CDAEDA1F0883AA106BC01AC66DC5133DB437DDAD81", {4, 6, 1, 3, 5, 0, 2}), 7: ("1B129292F2FD981AE5185302AF781D6AB9657E14C0B953279838C62EDF07A12A", {5, 0, 1, 3, 2, 6, 4}), 8: ("1650BAB553B0170F2AE5CDDD3308FC9A890FB833C48E8A9EFFE7E77C77C61E4C", {3, 1, 2, 0, 4, 6, 5})}
7
reason: all committed
done!
